src/Objs/Homepage/HomepageEditingConfig.ts

```diff
@@ -16,6 +16,11 @@ provideEditingConfig(Homepage, {
       title: 'Page description',
       description: 'Limit to 175, ideally 150 characters.',
     },
+    pisaUrl: {
+      title: 'PisaSales Portal API URL',
+      description:
+        'Only editable on the default site. Reload the app after changing this value.',
```

Comment:
* [ ] FYI how is the editor supposed to know which site is the "default site"? Maybe we should put the site's `contentTitle` here dynamically?
* [ ] FYI not sure if the average editor knows how to "reload the app". Maybe just remove this hint and assume a reload is the default action of any user if something doesn't work as expected?

Reply:
True. Other ideas:
* Change the text to "English homepage"
* Hide the option on other sites
* Allow editing and show a validation error